doc/developer/topotests.rst

```diff
@@ -680,6 +680,28 @@ during the config_timing test.
 To specify different arguments for ``perf record``, one can use the
 ``--perf-options`` this will replace the ``-g`` used by default.
 
+Running Daemons under RR Debug (``rr record``)
+""""""""""""""""""""""""""""""""""""""""""""""
+
+Topotest can automatically launch any daemon under ``rr(1)`` to collect
+execute state. The daemon is run in non-daemon mode with ``rr record``.
```

Comment:
execution*, and maybe "in the foreground" would be clearer than "non daemon mode"